src/core/components/ConfigModel.tsx

```diff
@@ -266,7 +291,11 @@ export default function ConfigModel({ handleOpen, handleClose, handleParentModal
                                     label={translate('CALLBACK') + ' ' + translate('URL')}
                                     placeholder={translate('CALLBACK') + ' ' + translate('URL')}
                                 />
-                                <Tooltip onMouseLeave={handleTooltipMouseLeave} onClick={() => handleCopyClick(`https://www.wavemakeronline.com/studio/services/oauth2/${providerConf?.providerId}/callback`)} sx={{ ":hover": { backgroundColor: 'transparent' } }} title={tooltipTitle}>
+                                <Tooltip onMouseLeave={handleTooltipMouseLeave} onClick={() => handleCopyClick(providerConf
+                                    ? `https://www.wavemakeronline.com/studio/services/oauth2/${providerConf.providerId}/callback`
```

Comment:
can we move the domain name to configuration?

Reply:
Yes Vivek. Changed domain name from configuration